libexec/scoop-depends.ps1

```diff
@@ -15,13 +15,13 @@ reset_aliases
 $opt, $apps, $err = getopt $args 'a:' 'arch='
 $app = $apps[0]
 
-if(!$app) { error 'ERROR: <app> missing'; my_usage; exit 1 }
+if(!$app) { error '<app> missing'; my_usage; exit 1 }
 
 $architecture = default_architecture
 try {
     $architecture = ensure_architecture ($opt.a + $opt.arch)
 } catch {
-    error "ERROR: $_"; exit 1
+    abort "ERROR: $_"
```

Comment:
This is fine, but kinda confusing. Could abort either not prefix 'ERROR: ', or use a different prefix, like 'FATAL: '?

Reply:
It could, but I wouldn't worry too much about it. As @r15ch13 said, the whole error handling needs to be changed anyway. The worst cases ("ERROR ERROR...") have been taken care of.

Reply:
Agreed. thx.